library/src/main/java/io/github/boguszpawlowski/composecalendar/states/CurrentState.kt

```diff
@@ -0,0 +1,38 @@
+package io.github.boguszpawlowski.composecalendar.states
+
+import androidx.compose.runtime.Stable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.saveable.Saver
+import androidx.compose.runtime.setValue
+import java.time.LocalDate
+
+@Suppress("FunctionName") // Factory function
+public fun CurrentState(initialDay: LocalDate): CurrentState = CurrentStateImpl(initialDay)
```

Comment:
I think `CurrentState` name is very vague. What does it represent? Is it a month / day / calendar state?